src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -30,7 +30,19 @@ private enum ParameterSyntaxKind : byte
 
         private CustomAttributesBag<CSharpAttributeData> _lazyCustomAttributesBag;
         private ThreeState _lazyHasOptionalAttribute;
-        protected ConstantValue _lazyDefaultSyntaxValue;
+        private ConstantValue _lazyDefaultSyntaxValue;
+
+        private Binder _parameterBinder
```

Comment:
Consider naming `LocalFunctionParameterBinder` so it's clear when the binder is expected.